packages/cli/src/bin/ceramic.ts

```diff
@@ -204,27 +204,30 @@ schemas
   })
 
 const pin = program.command('pin')
-pin.description(`('Ceramic local pinning API ${pc.red(pc.bold('[Deprecated]'))}`)
+pin.description(`Ceramic local pinning API ${pc.red(pc.bold('[Deprecated]'))}`)
```

Comment:
Should this really say deprecated?

Reply:
Probably not, I can remove this.
Shall I also remove the deprecation notice for other commands? IIRC we log a message about using the Glaze CLI, which we don't maintain anymore.

Reply:
Updated in new commit